lua/astrocommunity/pack/wgsl/init.lua

```diff
@@ -0,0 +1,23 @@
+local utils = require "astronvim.utils"
+return {
+  {
+    "szebniok/tree-sitter-wgsl",
+    dependencies = "nvim-treesitter/nvim-treesitter",
+    lazy = false,
+    build = "cp -r queries ../nvim-treesitter/queries/wgsl",
+    config = function()
+      vim.filetype.add { extension = { wgsl = "wgsl" } }
+      require("nvim-treesitter.parsers").get_parser_configs().wgsl = {
+        install_info = {
+          url = vim.fn.stdpath "data" .. "/lazy/tree-sitter-wgsl",
+          files = { "src/parser.c", "src/scanner.c" },
+        },
+      }
```

Comment:
This code should be moved into the `opts` function

Reply:
No this should be in the `config` function in pretty sure